multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -246,6 +256,29 @@ def _compute_rkd_loss(
 
         return loss
 
+    def _compute_softmax_regression_loss(
```

Comment:
Is there a paper for this loss type? It's kind of weird why distillation needs to compute loss in this way.

Reply:
Yes it's an iclr'21 paper: https://www.adrianbulat.com/downloads/ICLR2021/knowledge_distillation_via_softmax_regression_representation_learning.pdf